File: Observer/Java2/Acelerador.java
import java.util.ArrayList;
import java.util.List;
public class Acelerador implements SujetoObservable
{
    private List<Observador> observers;
    

    Acelerador()
    {
        this.observers =  new ArrayList<Observador>();
    }

    public void pisarAcelerador()
    {
        // Vamos a subir la potencia del motor
        this.notificar();
    }

    public void attach(Observador o)
    {
        this.observers.add(o);
    }

    @Override
    public void notificar()
    {
        for(Observador o: observers){
            o.update();
        }
    }
}

File: Observer/Python/Acelerador.py
from SujetoObservable import *
from Observador import *

class Acelerador(SujetoObservable):
    def __init__(self):
        self._observers = []
    
    def pisarAcelerador(self):
        self.notificar()
    
    def attach(self, o):
        self._observers.append(o)
    
    def notificar(self):
        for o in self._observers:
            o.update()